corehq/warehouse/dbaccessors.py

```diff
@@ -40,3 +42,24 @@ def _get_ids_by_last_modified(cls, doc_types, start_datetime, end_datetime):
         )
         for result in results:
             yield result['id']
+
+
+def get_synclog_ids_by_date(start_datetime, end_datetime):
+    from casexml.apps.phone.models import SyncLog
+
+    results = SyncLog.view(
+        "sync_logs_by_date/view",
+        startkey=[json_format_datetime(start_datetime)],
+        endkey=[json_format_datetime(end_datetime)],
+        reduce=False,
+        include_docs=False
+    )
+    for result in results:
+        yield result['id']
```

Comment:
yeah it depends on the etl process. if it's a dim it wouldn't matter,if it's a fact it might cause us to increment a count one too many or insert one too many records. might be good to just make it a standard that we always make start date exclusive and end date inclusive